googlemaps/distance_matrix.py

```diff
@@ -75,6 +75,10 @@ def distance_matrix(client, origins, destinations,
         requests. Valid values are "less_walking" or "fewer_transfers".
     :type transit_routing_preference: string
 
+    :param traffic_model: Specifies the predictive travel time model to use.
```

Comment:
Do you think it's worth calling out the restrictions here?

e.g. (from the docs): The traffic_model parameter may only be specified for requests where the travel mode is driving, and where the request includes a departure_time, and only if the request includes an API key or a Google Maps API for Work client ID.

(you can drop the last part, since it should always be true in the library's context)